source/features/rgh-linkify-yolo-issues.tsx

```diff
@@ -0,0 +1,27 @@
+import * as pageDetect from 'github-url-detection';
+
+import React from 'dom-chef';
+
+import select from 'select-dom';
+
+import features from '../feature-manager';
+
+import {wrap} from '../helpers/dom-utils';
+
+import {getRghIssueUrl} from '../helpers/rgh-issue-link';
+
+function init(): void {
+	for (const issueCell of select.all('td:nth-child(3)')) {
+		wrap(issueCell.firstChild!, <a href={getRghIssueUrl(issueCell.textContent!)}/>);
+	}
+}
+
+void features.add(import.meta.url, {
+	include: [
+		pageDetect.isRepo,
+		pageDetect.hasCode,
+		pageDetect.isSingleFile,
```

Comment:
We need a more specific custom function just to run this feature on the specific file (but on any branch)

Reply:
`() => new URLSearchParams(location.pathname).has('broken-features.csv')` i have added this function but i am not sure if this is what you expected.

Reply:
Almost! This doesn't work, pathname is not URLSearchParams. You can use location.pathname.endsWith(blablah.csv)